tracer/src/Datadog.AutoInstrumentation.NativeLoader/util.h

```diff
@@ -60,27 +52,6 @@ static fs::path GetConfigurationFilePath()
     }
 }
 
-static std::string GenerateRuntimeId()
-{
-#ifdef WIN32
-    UUID uuid;
-    UuidCreate(&uuid);
-
-    unsigned char* str;
-    UuidToStringA(&uuid, &str);
-
-    std::string s((char*) str);
-
-    RpcStringFreeA(&str);
-#else
-    uuid_t uuid;
-    uuid_generate_random(uuid);
-    char s[37];
-    uuid_unparse(uuid, s);
-#endif
-    return s;
-}
-
```

Comment:
I wonder, is it worth creating a separate PR for the changes to the runtime ID generation, just for clarity and to get extra eyes on it? It seems technically independent of the CI changes?

Reply:
The issue is that the alpine tests will fail.

Reply:
🤔 But you don't make any changes to the Docker image in this PR do you?

Reply:
We were talking at cross purposes - extracted to a separate PR #2792 here, thanks :)